django/website/logframe/tests/test_widgets.py

```diff
@@ -1,8 +1,17 @@
 from ..widgets import ColorChoiceInput
+from logframe.widgets import ColorSelect
 
 
 def test_color_choice_input_widget_rendering():
     color_choice_input = ColorChoiceInput('color_widget', 1, {'id': '1'}, ('blue', 'Blue'), 1)
     expected_output = '<input id="1_1" name="color_widget" type="radio" value="blue" /> <label for="1_1" class="color-tag blue">Blue</label>'
     actual_output = color_choice_input.render()
     assert expected_output == actual_output
+
+
+def test_color_choice_input_widget_not_escaped_when_rendering_color_select():
```

Comment:
This looks like quite a brittle test - the smallest change to the html (eg white space or a CSS class will cause it to fail. If all we're doing is making sure the output isn't escaped, can't we just check something smaller in the output? Or can we parse it with soupparser or something?